src/python.cc

```diff
@@ -798,6 +801,35 @@ ModelInstanceState::SetupChildProcess()
             .c_str());
   }
 
+  // Path to the extracted Python env
+  std::string python_execution_env = "none";
```

Comment:
Why not just leave the string empty to indicate it is not set/intialized?

Reply:
I fixed this.